config/config.go

```diff
@@ -1,11 +1,19 @@
 package config
 
 import (
+	"github.com/ONSdigital/dp-feedback-api/sdk"
 	"time"
 
 	"github.com/kelseyhightower/envconfig"
 )
 
+// Config for feedback sdk
+type FeedbackConfig struct {
+	BindAddr         string
```

Comment:
maybe this could be called `APIURL`, with the tag `envconfig:"FEEDBACK_API_URL"`

Reply:
Renamed BindAddr to APIURL